pokebot.py

```diff
@@ -44,20 +49,20 @@
 # For those cases, we register an `atexit` handler that will wait for user input before closing
 # the terminal window.
 def on_exit() -> None:
-    if platform.system() == "Windows":
-        import psutil
-        import os
+    """Windows-specific handler to prevent the terminal from being terminated until user input."""
+    import psutil
+    import os
+    parent_process_name = psutil.Process(os.getppid()).name()
+    if parent_process_name == "py.exe" or is_bundled_app():
+        if gui is not None and gui.window is not None:
+            gui.window.withdraw()
 
-        parent_process_name = psutil.Process(os.getppid()).name()
-        if parent_process_name == "py.exe" or is_bundled_app():
-            if gui is not None and gui.window is not None:
-                gui.window.withdraw()
-
-            print("")
-            input("Press Enter to close...")
+        print("")
+        input("Press Enter to close...")
 
 
-atexit.register(on_exit)
+if OS_NAME == "Windows":
```

Comment:
This is very minor but I didn't see the point of registering a function that did nothing on Linux.